swarm-production/docker-stack.yml

```diff
@@ -38,17 +39,21 @@ services:
           - node.labels.io.zenko.type == storage
 
   s3-front:
-    image: scality/s3server
+    image: zenko/cloudserver:pensieve-0
     ports:
-      - "8000"
+      - "8001"
     networks:
-      - backend
-      - frontend-dmz
+      backend:
+        aliases:
```

Comment:
why an alias?

Reply:
allows talking to s3 through the backend network with a known endpoint (errors out with 400 otherwise)